### ISSUE TITLE
Schema Compare for sql database projects in VSCode

### ISSUE THREAD

**Issue by alexszasz** [microsoft/azuredatastudio#18245]
I see in the feature list in the extension that there should be a command to compare the schema in a DB project (project created from SSDT) but I can't find the way to actually use it. 
Am I missing something? Or did I stumble on an edge case (Mac + VSCode + SQL Database Projects extension)?

**Comment by kisantia** [comment]
Schema compare is currently only available in ADS

**Comment by dzsquared** [comment]
Hi @alexszasz  - sorry for the confusing documentation combined between Azure Data Studio and VS Code SQL projects.  A few of the more graphical interfaces are only available in Azure Data Studio at this time.  Putting it on my list to clear that up next week.

**Comment by dzsquared** [comment]
@alexszasz schema compare for sql projects in ADS has been available for all of the ADS platforms, including macOS, since the sql projects extension was added in summer 2020.  Please let us know if it does not appear in your extension gallery.

![image](https://user-images.githubusercontent.com/12227241/153907651-7d354779-4a61-4bfb-9d01-9a527bd36628.png)

**Comment by alexszasz** [comment]
Thanks. I installed it now and it's there, but I think some features are still missing which would make it really hard for me to use, so I might stick to the windows SSDT for a while.

Thanks again for all your help.

**Comment by kisantia** [comment]
closing this issue as there aren't plans to add schema compare in vscode in the near future

**Comment by ko1oski** [comment]
Hi, i'm looking for an alternative to using Visual Studio Schema Compare and I found this closed issue. I'm very interested if there are any plans or items in the backlog regarding this feature. I've used ADS recently and Schema Compare these is for basic purposes only as it won't let me display a context menu, which means I can't select all and uncheck. This leads to necessity of unchecking 100s or 1000s of objects in big projects. Not to mention the fact that object list is living its own life and can reset itself to a specific view, probably triggered by some live dependency check running in the background after each object is deselected. A nightmare to use for someone who worked with Visual Studio for a while.
